src/components/backend-ai-console.ts

```diff
@@ -716,6 +721,22 @@ export default class BackendAIConsole extends connect(store)(LitElement) {
       document.dispatchEvent(event);
     }
   }
+  _readRecentProjectGroup() {
+    let value: string | null = localStorage.getItem('backendaiconsole.projectGroup');
+    if (value !== null && value !== '' && value !== '""') {
+      return value;
+    } else {
+      return window.backendaiclient.current_group;
+    }
+  }
+
+  _writeRecentProjectGroup(value) {
+    if (value !== null && value !== '' && value !== '""') {
```

Comment:
here as well.

Reply:
I understand. It could be simplified, too. :)